reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+---
+ms.date:  2017-12-04
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+title:  about_Logging
+---
+
+# About Logging
+
+## Short Description
+
+PowerShell logs details of PowerShell operations, such as starting
+and stopping the program engine and starting and stopping PowerShell
+providers.  it will also log details about PowerShell commands.
+
+## Long Description
+
+The location of PowerShell logs is dependent on the target platform.
+On Windows, PowerShell logs to the event log, on Linux, PowerShell logs
+to syslog, and on MacOS, the os_log logging system is used.
+
+## Viewing the PowerShell event log on Windows
+
+PowerShell logs can be viewed using the Event Viewer. The event log is
+located in the Application and Services Logs group and is named PowerShellCore.
+The associated ETW provider guid is `{f90714a8-5509-434a-bf6d-b1624c8a19a2}'
```

Comment:
For markdown, need to open and close with backtick to highlight that text.  You used an apostrophe on the end which messes up the parsing.

Reply:
fixed